[8-4] 다음과 같은 메서드가 있을 때, 예외를 잘못 처리한 것은? (모두 고르시오)

c. try {method();} catch(Exception e) {} catch(NumberException e) {}
Exception은 catch문의 가장 마지막에 와야 함
